src/Rating.jsx

```diff
@@ -100,7 +101,8 @@ class Rating extends React.PureComponent {
       direction,
       emptySymbol,
       fullSymbol,
-      placeholderSymbol
+      placeholderSymbol,
+      ...other
```

Comment:
Please remove passing other props. Let's open an issue to discuss any specific additions you'd like to see to the API. I don't really like the idea of mass passing of undocumented props.